probe-rs/src/bin/probe-rs/cmd/complete.rs

```diff
@@ -267,9 +267,22 @@ impl ShellExt for Bash {
 }
 
 impl ShellExt for PowerShell {
-    fn install(&self, _file_name: &str, script: &str) -> Result<()> {
-        eprintln!("Install the script in location of your choice and run it with `Import-Module completion-script.ps1`");
-        println!("{script}");
-        Ok(())
+    fn install(&self, file_name: &str, script: &str) -> Result<()> {
+        let Some(dir) = directories::UserDirs::new() else {
+            eprintln!("The user home directory could not be located.");
+            eprintln!("Install the script in ~\\Documents\\WindowsPowerShell\\{file_name}");
+            println!("{script}");
+            return Ok(());
+        };
+        let path = dir
+            .home_dir()
+            .join("Documents")
+            .join("WindowsPowerShell")
+            .join(file_name);
+        eprintln!(
+            "Install the script in location of your choice and run it with `Import-Module {}`",
```

Comment:
I find it a bit funny that if the script does NOT write the file, we say "a location of your choice", but if we just dump it we specify a location. Shouldn't it be backwards?

Reply:
Fixed :)